tests/state_test.go

```diff
@@ -61,6 +68,7 @@ func TestState(t *testing.T) {
 	for _, dir := range []string{
 		stateTestDir,
 		legacyStateTestDir,
+		benchmarksDir, // FIXME: This does not seem to work, but we want to test benchmarks!
```

Comment:
What is this fixme about? Do the tests in the benchmarks dir still not work? Something we need to look into or can this comment be removed?

Reply:
I believe it actually works as expected. Let me find it in the logs...

Reply:
These run, however the "benchmark" dir name is omitted.
```
--- PASS: TestState (0.05s)
    --- PASS: TestState/main/weierstrudel.json (0.05s)
        --- PASS: TestState/main/weierstrudel.json/London/0/trie (0.00s)
        --- PASS: TestState/main/weierstrudel.json/London/0/snap (0.00s)
        --- PASS: TestState/main/weierstrudel.json/London/1/trie (0.00s)
        --- PASS: TestState/main/weierstrudel.json/London/1/snap (0.00s)
        --- PASS: TestState/main/weierstrudel.json/London/2/trie (0.00s)
        --- PASS: TestState/main/weierstrudel.json/London/2/snap (0.00s)
        --- PASS: TestState/main/weierstrudel.json/London/3/trie (0.01s)
        --- PASS: TestState/main/weierstrudel.json/London/3/snap (0.01s)
        --- PASS: TestState/main/weierstrudel.json/London/4/trie (0.01s)
        --- PASS: TestState/main/weierstrudel.json/London/4/snap (0.01s)
    --- PASS: TestState/main/blake2b_huff.json (0.17s)
```